conf/config.go

```diff
@@ -24,11 +23,12 @@ type Server struct {
 
 // TiKV config is the config of tikv sdk
 type TiKV struct {
-	PdAddrs string `cfg:"pd-addrs;required; ;pd address in tidb"`
-	GC      GC     `cfg:"gc"`
-	Expire  Expire `cfg:"expire"`
-	ZT      ZT     `cfg:"zt"`
-	TiKVGC  TiKVGC `cfg:"tikv-gc"`
+	PdAddrs string     `cfg:"pd-addrs;required; ;pd address in tidb"`
+	GC      GC         `cfg:"gc"`
+	Expire  Expire     `cfg:"expire"`
+	ZT      ZT         `cfg:"zt"`
+	TiKVGC  TiKVGC     `cfg:"tikv-gc"`
+	TiKVLog TiKVLogger `cfg:"tikv-logger"`
```

Comment:
```suggestion
	TiKVLog TiKVLogger `cfg:"logger"`
```
I think `logger` is enough as the name.